ec2/spark_ec2.py

```diff
@@ -591,11 +591,15 @@ def launch_cluster(conn, opts, cluster_name):
 
     # AWS ignores the AMI-specified block device mapping for M3 (see SPARK-3342).
     if opts.instance_type.startswith('m3.'):
+        if sys.version_info[0] >= 3:
+            letters = string.ascii_letters
```

Comment:
Actually, it looks like Python 2.6 also has `string.ascii_letters`: https://docs.python.org/2.6/library/string.html#string.ascii_letters

Why not just use that for both Python 2.x and 3.x so that we don't need this conditional check?